src/components/HotelDetails/HotelDetails.jsx

```diff
@@ -0,0 +1,144 @@
+import { Link, useLocation, useNavigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { Rating } from '@mui/material';
+import { removeHotel } from '../../redux/hotels/hotels';
+import './HotelDetails.css';
+
+/* eslint linebreak-style: ["error", "windows"] */
+const HotelDetails = () => {
+  const location = useLocation();
+  const hotel = location.state;
+  const navigate = useNavigate();
+  console.log(location);
```

Comment:
- [ ] Kindly remove this old piece of debugging console.log statement, in order to improve the repo's cleanliness. 👍